runtime/src/accounts_db.rs

```diff
@@ -165,6 +166,21 @@ impl AccountStorageEntry {
     }
 }
 
+pub struct ParThreadPool {
```

Comment:
why is this in accounts_db?  why is there a wrapper?

Reply:
Wrapper is for default(). Bank and AccountsDB need a default implementation.

Reply:
Would it be better if it were in its own file?

Reply:
Removed